README.md

```diff
@@ -226,6 +230,25 @@ Use the virtual camera with `OBS Studio`:
 - Click "Start Virtual Camera" button in the controls section
 - Select "OBS Cam" as default camera in the video settings of the application target of the injection
 
+## Run dot with an Android emulator

```

Comment:
can someone else test this first? @Ghassen-Chaabouni maybe?

Reply:
I added these steps. I already tested them